src/api/spec/features/beta/webui/groups_spec.rb

```diff
@@ -28,6 +28,8 @@ def group_in_datatable(page, group)
   scenario 'visit group show page' do
     visit group_show_path(group_1)
 
+    # TODO: Remove this line if the dropdown is changed to a scrollable tab
```

Comment:
Guess you want to mention `responsive_ux` so we find this once we move this out of beta...

Reply:
Added a mention to `responsive_ux`.